components/authz-service/server/v2/policy_test.go

```diff
@@ -3209,7 +3227,13 @@ type testEngine struct {
 }
 
 func (te *testEngine) V2SetPolicies(
-	_ context.Context, policies map[string]interface{},
+	ctx context.Context, policies map[string]interface{},
+	roles map[string]interface{}, rules map[string][]interface{}) error {
+	return te.V2p1SetPolicies(ctx, policies, roles, rules)
```

Comment:
Perhaps add a comment explaining why it is OK to use the same test code for v2 and v2p1 here.